tutorials/optimize/opt_gemm.py

```diff
@@ -49,8 +49,12 @@
 # Random generated tensor for testing
 a = tvm.nd.array(numpy.random.rand(N, N).astype(dtype), tvm.cpu(0))
 b = tvm.nd.array(numpy.random.rand(N, N).astype(dtype), tvm.cpu(0))
+
+now = time.time()
 # The expected answer
 answer = numpy.dot(a.asnumpy(), b.asnumpy())
+np_runing_time = time.time() - now
+print("Numpy running time: %f" % np_runing_time)
```

Comment:
Probably we want to use `timeit` for a better time measurement

Reply:
Changed